localization/english/wc_buildings_l_english.yml

```
@@ -1,11 +1,55 @@
+ building_temple_of_atal_hakkar_01:1 "Temple of Atal'Hakar"
+ building_temple_of_atal_hakkar_01_desc:1 " An influential group of troll priests, known as the Atal'ai, seek to bring back an ancient blood god named Hakkar the Soulflayer. They erected this great temple to Hakkar - where they could prepare for his arrival into the physical world."
+ 
+ building_type_thoradins_wall_01:0 "Thoradin's Wall"
+ building_type_thoradins_wall_01_desc:1 "Colossal walls that sheild the denzins of its holders domain. All unwanted guests must firstly face this almost insurmountable obstacle."
+ building_thoradins_wall_01:1 "Ruined Thoradin's Wall"
+ building_thoradins_wall_01_desc:1 "Once a wonder for our people, a testament to our power and our technological prowess, Thoradin's Wall now only statnds as a symbol of our downfall. Some parts of the wall still remain and help keep the peace, but the mostly ruined wall is but a shell of the once great shield against those who would wish harm upon us."
+ building_thoradins_wall_02:1 "Thoradin's Wall"
+ building_thoradins_wall_02_desc:1 "In its full glory, Thoradin's Wall stands proudly as a bastion for our people. We are protected by the towering walls and the brave soldiers guarding us from the dangers outside."
+ 
+ building_type_ruined_palace_01:0 "Ruined Palace"
+ building_type_ruined_palace_01_desc:1 "A city that spans beneath much of Lordaeron's ruin. A grave to kingdom."
+ building_ruined_palace_01:1 "Ruined Palace"
+ building_ruined_palace_01_desc:1 "The Ruins of Lordaeron is all that remains of the city of Lordaeron, the great capital city of the Kingdom of Lordaeron. Now it serves as a sad reminder of the power of the Scourge. The Ruined Palace still stands and its inhabitants are constantly haunted by the final tragic events that led to the death of King Terenas and the city's destruction."
+ 
+ building_type_sunwell_01:0 "The Sunwell"
+ building_type_sunwell_01_desc:1 "A fountain of mystical power located in the Sunwell Plateau on the Isle of Quel'Danas. Its presence in its users lives is comparable to eating or breathing."
+ building_sunwell_01:1 "The Sunwell"
+ building_sunwell_01_desc:1 "The Sunwell has empowered us high elves for roughly seven thousand years. Its presence in their our is comparable to eating or breathing. In the Sunwell's light, Quel'Thalas has grown mighty, becoming a beacon of power in the Eastern Kingdoms."
+
+ building_type_violet_citadel_01:0 "Violet Citadel"
+ building_type_violet_citadel_01_desc:1 "A Citadel working as both a place of study and progress as well as a bastion houseing great casters to defend against invading forces."
+ building_violet_citadel_01:1 "Violet Citadel"
+ building_violet_citadel_01_desc:1 "The Violet Citadel is the main fortress of Dalaran which houses the Chamber of Air, research laboratories and great spell libraries of the Kirin Tor."
```

```suggestion
 building_violet_citadel_01_desc:1 "The Violet Citadel is the main fortress of Dalaran which houses the Chamber of Air, research laboratories, and the great spell libraries of the Kirin Tor."
```

I would add an Oxford comma here for readability. 😄